app/assets/stylesheets/main.scss

```diff
@@ -13,8 +13,7 @@
 @import "layouts/user_header";
 
 // Pages CSS
-@import "pages/home";
```

Comment:
any reason why we are removing `pages/home`?

Reply:
@sanghun89 yeah, so originally about.scss was home.scss, but @gcwokeup and I decided that it made more sense to separate the about page css from the index page css. so we updated the main.css to say pages/about instead of pages/home. A little tricky to explain via comments so let me know if that makes sense.